man/systemd.netdev.xml

```diff
@@ -639,6 +639,30 @@
         </listitem>
       </varlistentry>
       <varlistentry>
+        <term><varname>Key=</varname></term>
+        <listitem>
+          <para>The <literal>Key=</literal> parameter specifies the same key to use in
+          both directions (<literal>InputKey=</literal> and <literal>OutputKey=</literal>).
+          The <literal>Key=</literal> is either a number or an IPv4 address-like dotted quad.
+          It is used as mark-configured SAD/SPD entry as part of the lookup key (both in data
+          and control path) in ip xfrm. It is only used for VTI/VTI6 tunnels.</para>
```

Comment:
well, if that's the case then this should be in the man page of course, not here in the comments.